src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/index.tsx

```diff
@@ -136,32 +219,29 @@ class IssueRuleEditor extends AsyncView<Props, State> {
 
     addLoadingMessage();
 
-    try {
-      const resp = await this.api.requestPromise(endpoint, {
-        method: isNew ? 'POST' : 'PUT',
-        data: rule,
-      });
-      this.setState({detailedError: null, rule: resp});
-
-      // The onboarding task will be completed on the server side when the alert
-      // is created
-      updateOnboardingTask(null, organization, {
-        task: OnboardingTaskKey.ALERT_RULE,
-        status: 'complete',
-      });
-
-      addSuccessMessage(isNew ? t('Created alert rule') : t('Updated alert rule'));
-
-      // When editing, there is an extra route to move back from
-      const stepBack = isNew ? -1 : -2;
-      browserHistory.replace(recreateRoute('', {...this.props, stepBack}));
-    } catch (err) {
-      this.setState({
-        detailedError: err.responseJSON || {__all__: 'Unknown error'},
-        loading: false,
-      });
-      addErrorMessage(t('An error occurred'));
-    }
+    this.api.request(endpoint, {
```

Comment:
I removed the async/await because we need to know whether the response code is a `200` or a `202` and if it's the latter then we need to hit the rule task endpoint

Reply:
@MeredithAnya You can still keep `async/await` if you use the `includeAllArgs` option. See: https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/utils/projects.tsx#L430